extensions/multiline.md

```diff
@@ -54,6 +54,8 @@ The only defined capability key so far is:
 * `max-bytes` - This defines the maximum allowed total byte length of multiline batched content *REQUIRED*
 * `max-lines` - This defines the maximum allowed number of lines in a multiline batch content *RECOMMENDED*
```

Comment:
Suggestion: "This defines the maximum allowed number of lines in a multiline batch"